test/transformers/test_tvd.py

```diff
@@ -0,0 +1,132 @@
+from test.utils import supports_bfloat16
+
+import pytest
+import torch
+
+from liger_kernel.transformers.tvd import LigerTVDLoss
+
+
+class TorchTVDLoss(torch.nn.Module):
```

Comment:
I feel it'll be very helpful if we can add ignore index along with this PR to make TVD complete, similar to how JSD is doing it -- https://github.com/linkedin/Liger-Kernel/blob/main/src/liger_kernel/ops/jsd.py

Reply:
+1 which would be very helpful to cover broader use cases

Reply:
done!